ts/components/game/Player.tsx

```diff
@@ -42,6 +42,73 @@ class Player extends React.Component<PlayerProps, {}> {
 			};
 		};
 
+		var activatedHeroPower = false;
```

Comment:
Need to fix this throughout the entire codebase :P. I always forget and just take whatever I see closest.